### PR TITLE
Product list

### PR DESCRIPTION
**Pull request by md-amir-gauhar**
--added filter component
--added card component and product listing page
Screenshot (194)

### CODE REVIEW

File: src/components/Filter.jsx [hunk 1]
```diff
@@ -0,0 +1,171 @@
+import React, { useState, useEffect } from 'react'
+import axios from 'axios'
+import { FaFilter } from 'react-icons/fa'
+
+import '../styles/Filter.css'
+
+const Filter = ({mnag, setManga}) => {
+  const [isChecked, setIsChecked] = useState({
+    lowtohigh: false,
+    hightolow: false,
```
Comment (Pratik1005) [inline src/components/Filter.jsx]: Just as suggestion: Be consistent with variable naming, if you are using camelCase then stick with it.

File: src/components/Filter.jsx [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+          <span>SORT</span>
+          <div>
+            <input
+              type="radio"
+              name='low'
+              value='low'
+              checked={isChecked.lowtohigh}
+              onChange={radioHandler}
+            />
+            <label>Low to High</label><br />
```
Comment (Pratik1005) [inline src/components/Filter.jsx:66]: In the label, you can use htmlFor attribute so when we click on label the corresponding radio input will be checked

**Review by Pratik1005 — APPROVED** [review]
Other than suggested changes, rest looks good

**Review by Hamza-777 — APPROVED** [review]
Looks good.

**Pull request merged**